src/wechaty-manager.ts

```diff
@@ -270,7 +270,7 @@ export class WechatyManager extends Manager {
     log.verbose('WechatyManager', 'onMessage("%s") from "%s" to "%s" with age "%s" (timestamp: "%s")',
       message,
       message.from()!.id,
-      message.to()!.id,
```

Comment:
It will throw error like follow when a user send message to a room.
```
TypeError: Cannot read property 'id' of null
    at WechatyManager.onMessage (/home/ylin/Documents/matrix-appservice-wechaty/src/wechaty-manager.ts:273:20)
    at Wechaty.onMessage (/home/ylin/Documents/matrix-appservice-wechaty/src/wechaty-manager.ts:65:50)
    at Wechaty.emit (events.js:315:20)
    at Wechaty.EventEmitter.emit (domain.js:467:12)
    at PuppetPadlocal.<anonymous> (/home/ylin/Documents/matrix-appservice-wechaty/node_modules/wechaty/src/wechaty.ts:500:20)
```